app/build.gradle

```diff
@@ -65,7 +65,16 @@ dependencies {
     testImplementation 'org.jetbrains.kotlinx:kotlinx-coroutines-test:1.6.0'
     implementation 'com.squareup.retrofit2:retrofit:2.7.1'
     implementation 'com.squareup.retrofit2:converter-gson:2.7.1'
+
     // Koin
     implementation "io.insert-koin:koin-android:3.3.3"
     implementation 'com.squareup.okhttp3:okhttp:4.10.0'
+    implementation 'io.mockk:mockk:1.12.2'
+
+    allprojects {
+        configurations.all {
+            resolutionStrategy.force 'org.objenesis:objenesis:2.6'
+        }
+    }
```

Comment:
What is this?

Reply:
I had an error 
MethodHandle.invoke and MethodHandle.invokeExact are only supported starting with Android O (--min-api 26): Lorg/objenesis/instantiator/util/DefineClassHelper$Java11;defineClass(Ljava/lang/String;[BIILjava/lang/Class;Ljava/lang/ClassLoader;Ljava/security/ProtectionDomain;)Ljava/lang/Class;
and I find the solution on the internet. My app still has this error if I don't have this code.